engine/runtime/src/main/java/org/enso/interpreter/instrument/job/VisualizationResult.java

```diff
@@ -0,0 +1,36 @@
+package org.enso.interpreter.instrument.job;
+
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+import java.nio.charset.StandardCharsets;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.interpreter.runtime.error.WithWarnings;
+
+public final class VisualizationResult {
+  private VisualizationResult() {
+  }
+
+  public static byte[] visualizationResultToBytes(Object value) {
+    if (value instanceof byte[] arr) {
+      return arr;
+    }
+    if (value instanceof String text) {
+      return text.getBytes(StandardCharsets.UTF_8);
+    }
+    if (value instanceof Text text) {
+      return visualizationResultToBytes(text.toString());
+    }
+    if (value instanceof WithWarnings warn) {
+      return visualizationResultToBytes(warn.getValue());
+    }
+    var iop = InteropLibrary.getUncached();
```

Comment:
Not sure how `getUncached()` works. Is it worth making it a static member of the class?

Reply:
This is the code:
```
    public static InteropLibrary getUncached() {
        return UNCACHED;
    }
```
e.g. it already returns static final field.